p4-16/spec/P4-16-spec.mdk

```diff
@@ -6201,6 +6197,12 @@ in actions invoked from tables `flowlet` (reading) and `new_flowlet`
 would not execute atomically: a second packet may read the state of `r`
 before the first packet had a chance to update it.
 
+Note that even within an `action` definition, if the action does
+something like reading a register, modifying it, and writing it back,
+in a way that only the modified value should be visible to the next
+packet, then that portion of the action definition should be enclosed
```

Comment:
... then, to guarantee correct execution in all cases, that portion ...

Reply:
Done.